src/components/aside/Aside.js

```diff
@@ -1,7 +1,63 @@
 import React from 'react';
 
+import { Link } from 'react-router-dom';
+import './Aside.scss';
+
 const Aside = () => {
-  return <div>무지막지</div>;
+  return (
+    <div className="aside">
+      {CATEGOTY_LIST.map(category => {
+        return (
+          <div className="asideCategory" key={category.id}>
+            <h2 className="categoryName">
+              <Link to="/mens">{category.name}</Link>
+            </h2>
+            <ul className="categoryTypes">
+              {category.types.map(types => {
```

Comment:
```suggestion
              {category.types.map(type => {
```

types 배열 안에 하나의 아이템들은 type이라고 부르는게 이치에 맞아보이네요

사소한 변수명부터시작해서 전체 코드의 가독성이 시작됩니다!

항상 오해없도록, 의미가 잘 드러나도록 이름 지어주세요!

Reply:
이부분도 다른팀원이 수정 하겠다고 해서 다른 팀원이 수정 후 올리겠습니다